### DISCUSSION TITLE
Many errors with compiling with 1.0.2

### DISCUSSION THREAD

**Discussion by ashaogroq** [capnproto/capnproto#2113] [General]
`/nix/store/x58iykqzbmbzjjk47ljg2brjnw68mkj3-capnproto-1.0.2/include/capnp/any.h:1081:22: error: 'AnyList' was not declared in this scope; did you mean 'capnp::AnyList'?
 1081 | struct OrphanGetImpl<AnyList, Kind::OTHER> {
      |                      ^~~~~~~
      |                      capnp::AnyList`

New to working with capnproto and serializers in general. Have version 1.0.2 added as a package and building with nix, running into a lot of errors that seem to do with the files being included in my generated .hpp files (a script renames all `.capnp.h` files to `.capnp.hpp`, same with `.c++`. Any ideas on what could be causing this?

**Comment by kentonv** [comment]
It looks like the code has become confused about what namespace it is in. But I have no idea how that might have happened. It doesn't seem like the file names should make a difference.
